getting-started/openshift/installation.md

```diff
@@ -1,52 +1,124 @@
+
+```bash
+openshift-install create manifests
 ```
-[OSEv3:children]
-masters
-nodes
-etcd
 
-[OSEv3:vars]
-os_sdn_network_plugin_name=cni
-openshift_use_calico=true
-openshift_use_openshift_sdn=false
+Download the {{site.prodname}} manifests for OpenShift and add them to the generated manifests directory:
+
+```bash
+curl {{ "/manifests/ocp/crds/01-crd-installation.yaml" | absolute_url }} -o manifests/01-crd-installation.yaml" | absolute_url }}
+curl {{ "/manifests/ocp/crds/01-crd-tigerastatus.yaml -o manifests/01-crd-tigerastatus.yaml" | absolute_url }}
+curl {{ "/manifests/ocp/tigera-operator/00-namespace-tigera-operator.yaml" | absolute_url }} -o manifests/00-namespace-tigera-operator.yaml
+curl {{ "/manifests/ocp/tigera-operator/02-rolebinding-tigera-operator.yaml" | absolute_url }} -o manifests/02-rolebinding-tigera-operator.yaml
+curl {{ "/manifests/ocp/tigera-operator/02-role-tigera-operator.yaml" | absolute_url }} -o manifests/02-role-tigera-operator.yaml
+curl {{ "/manifests/ocp/tigera-operator/02-serviceaccount-tigera-operator.yaml" | absolute_url }} -o manifests/02-serviceaccount-tigera-operator.yaml
+curl {{ "/manifests/ocp/tigera-operator/02-tigera-operator.yaml" | absolute_url }} -o manifests/02-tigera-operator.yaml
+curl {{ "/manifests/ocp/01-cr-installation.yaml" | absolute_url }} -o manifests/01-cr-installation.yaml
+```
+
+> **Note**: The Tigera operator manifest downloaded above includes an initialization container which configures Amazon AWS
+> security groups for {{site.prodname}}. If not running on AWS, you should remove the init container from `manifests/02-tigera-operator.yaml`.
+{: .alert .alert-info}
+
+#### Add an image pull secret
+
+1. Download the pull secret manifest template into the manifests directory.
+
+   ```
+   curl {{ "/manifests/ocp/02-pull-secret.yaml" | absolute_url }} -o manifests/02-pull-secret.yaml
+   ```
 
-[masters]
-master1
+1. Update the contents of the secret with the image pull secret provided to you by Tigera.
 
-[nodes]
-node1
+   For example, if the secret is located at `~/.docker/config.json`, run the following commands.
 
-[etcd]
-etcd1
+   ```bash
+   SECRET=$(cat ~/.docker/config.json | tr -d '\n\r\t ' | base64 -w 0)
+   sed -i "s/SECRET/${SECRET}/" manifests/02-pull-secret.yaml
+   ```
+
+#### Create the cluster
+
+Start the cluster creation with the following command and wait for it to complete.
+
+```bash
+openshift-install create cluster
 ```
 
-You are now ready to execute the ansible provision which will install {{site.prodname}}. Note that by default,
-{{site.prodname}} will connect to the same etcd that OpenShift uses, and in order to do so, will distribute etcd's
-certs to each node. If you would prefer Calico not connect to the same etcd as OpenShift, you may modify the install
-such that Calico connects to an etcd you have already set up by following the [dedicated etcd install guide](dedicated-etcd).
+You can now monitor progress with the following command:
```

Comment:
I wish we gave some better info here. Don't they need to wait for Openshift to provision the cluster first? And then once kube-apiserver is running and has installed our CRDs, they can run the `watch` command, right? And also we don't tell them how they'll know when it has finished progressing. Maybe because it's so intuitive we don't have to explicitly say "once all statuses are Available"?